README.md

```diff
@@ -1,15 +1,16 @@
 # .NET Runtime - Managed Zlib
 
-This branch contains a template for standalone experiments, which means that experiments that are a library, which doesn't depend on runtime changes. This minimal template allows such experiments to avoid the overhead of having all the runtime, libraries and installer code.
+This is an experimental managed version, based on madler/zlib and deflate64 project, of ZLib's library, for exploring the idea of owning a competitive compression library that would follow the same cross-platform vision as .NET while having more consistent performance and compression characteristics. The project includes [Deflate algorithm](https://www.zlib.net/feldspar.html)'s implementation for raw inflate, based on [Deflate64](https://github.com/dotnet/runtime/tree/main/src/libraries/System.IO.Compression/src/System/IO/Compression/DeflateManaged), and raw deflate, based on [Zlib](https://github.com/madler/zlib), with a test harness for performance and functionality.
 
-This branch contains an experimental porting and comparison of the C Mark Adler's Zlib library used in System.IO.Compression to a C#. This is an exploration on whether we would benefit from switching our dependencies from madler/zlib to an own managed version. 
-The evaluation of this experiment will be based on performance information using dotnet benchmark and madler/zlib  as baseline,  to see the main areas of opportunity of performance improvement. Because the idea is to implement a managed version, porting madler/zlib, without losing functionality in the way, this experiment covers a set of unit tests. It is necessary to have enough of the design for the information gathered to be reliable. 
+Consuming an external dependency, like [Zlib](https://github.com/madler/zlib) or [Zlib-intel](https://github.com/intel/zlib), involves high-maintenance cross-platform and .NET versions, being shipped differently for Windows and Linux systems, and a vulnerability when it comes to fix reported issues or bugs. This library is being consumed by compression solutions like [DeflateStream](https://learn.microsoft.com/en-us/dotnet/api/system.io.compression.deflatestream?view=net-7.0), [GZipStream](https://learn.microsoft.com/en-us/dotnet/api/system.io.compression.gzipstream?view=net-7.0) , [ZLibStream](https://learn.microsoft.com/en-us/dotnet/api/system.io.compression.zlibstream?view=net-7.0) and [ZipArchive](https://learn.microsoft.com/en-us/dotnet/api/system.io.compression.ziparchive?view=net-7.0), among others.
```

Comment:
```suggestion
Consuming an external dependency like [Zlib](https://github.com/madler/zlib) or [Zlib-intel](https://github.com/intel/zlib) in .NET is a high maintenance task, since it involves shipping different versions for Windows and Unix systems, as well as having to either wait for vulnerabilities to get fixed upstream when they become public, or having to fix them ourselves, causing our code to potentially diverge from upstream.

This library is being consumed by compression solutions like [DeflateStream](https://learn.microsoft.com/en-us/dotnet/api/system.io.compression.deflatestream?view=net-7.0), [GZipStream](https://learn.microsoft.com/en-us/dotnet/api/system.io.compression.gzipstream?view=net-7.0) , [ZLibStream](https://learn.microsoft.com/en-us/dotnet/api/system.io.compression.zlibstream?view=net-7.0) and [ZipArchive](https://learn.microsoft.com/en-us/dotnet/api/system.io.compression.ziparchive?view=net-7.0), among others.
```